components/sinks/cu_zenoh_ros_sink/src/error.rs

```diff
@@ -0,0 +1,10 @@
+use cu29::CuError;
+use zenoh::Error as ZenohError;
+
+pub fn cu_error(msg: &str, error: ZenohError) -> CuError {
```

Comment:
Ok those ones are getting ridiculous, I am adding a note to refactor CuError with something like:
```Rust
#[derive(Debug)]
pub struct CuError {
    message: String,
    cause: Option<Box<dyn std::error::Error + Send + Sync>>,
}
```
No action needed on your part yet :)
https://github.com/copper-project/copper-rs/issues/317